openquake/server/views.py

```diff
@@ -376,7 +394,7 @@ def get_traceback(request, calc_id):
 
 @cross_domain_ajax
 @require_http_methods(['GET'])
-def get_result(request, result_id):
+def get_result_ng(request, calc_id, result_id):
     """
     Download a specific result, by ``result_id``.
 
```

Comment:
This breaks a test: see https://ci.openquake.org/job/zdevel_oq-engine/1149/testReport/junit/openquake.server.tests.functional_test/EngineServerTestCase/test_ok/